tests/wasm/0setup_test.go

```diff
@@ -0,0 +1,227 @@
+package wasm
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/chromedp/cdproto/cdp"
+	"github.com/chromedp/chromedp"
+)
+
+var dockerHeadless = flag.Bool("docker-headless", false, "Launch headless Chrome via docker instead looking for it on local filesystem")
+var noCleanup = flag.Bool("no-cleanup", false, "Skip cleanup operations (useful for manual viewing and debugging)")
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+
+	setup()
+	ret := m.Run()
+	cleanup()
+	os.Exit(ret)
+}
+
+func setup() {
+
+	// copy wasm_exec.js from targets to this dir
+	b, err := ioutil.ReadFile("../../targets/wasm_exec.js")
+	must(err)
```

Comment:
It may be a good idea to add an explanation to `must` calls, such as:

    must(err, "reading wasm_exec.js failed")

You can then print it together in the panic.